ydb/core/tx/replication/service/transfer_writer.cpp

```diff
@@ -498,12 +498,18 @@ class TTransferWriter
             return;
         }
 
+        if (entry.Status == TNavigate::EStatus::PathNotTable) {
+            return LogCritAndLeave("Only column tables are supported as transfer targets.");
+        }
+
         if (!CheckEntrySucceeded(entry)) {
             return;
         }
 
         // TODO support row tables
-        CheckEntryKind(entry, TNavigate::KindColumnTable);
+        if (entry.Kind != TNavigate::KindColumnTable) {
+            return LogCritAndLeave("Only column tables are supported as transfer targets.");
+        }
```

Comment:
```suggestion
        if (CheckEntryKind(entry, TNavigate::KindColumnTable);) {
            LOG_C("Only column tables are supported as transfer targets");
            return;
        }
```

Лог `CheckEntryKind()` может быть не настолько "understandable", но в нём больше полезной инфы.

Reply:
Как вы собираетесь показывать записи из логов пользователям?